src/Core/WinSWCore/XMLSchema.xsd

```diff
@@ -0,0 +1,391 @@
+          <xs:annotation>
+            <xs:documentation xml:lang="en" source="https://docs.microsoft.com/windows/win32/secauthz/security-descriptor-definition-language">
+              The security descriptor string for the service in SDDL form. See more details on SDDL on source link.
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
+
+        <xs:element name="priority" type="xs:string" minOccurs="0" maxOccurs="unbounded">
+          <xs:annotation>
+            <xs:documentation xml:lang="en">
+              Optionally specify the scheduling priority of the service process (equivalent of Unix nice) 
+              Possible values are idle, belownormal, normal, abovenormal, high, realtime (case insensitive.)
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
+
+        <xs:element name="stopparentprocessfirst" type="xs:string" minOccurs="0" maxOccurs="unbounded">
+          <xs:annotation>
+            <xs:documentation xml:lang="en">
+              Optionally specify the order of service shutdown. If true, the parent process is shutdown first. 
+              This is useful when the main process is a console, which can respond to Ctrl+C command and will 
+              gracefully shutdown child processes.
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
+
+        <xs:element name="startmode" type="xs:string" minOccurs="0" maxOccurs="unbounded">
+          <xs:annotation>
+            <xs:documentation xml:lang="en" source="https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/changestartmode-method-in-class-win32-service">
+              This element specifies the start mode of the Windows service. 
+              It can be one of the following values: Boot, System, Automatic, or Manual.
+              The default value is Automatic.
+              For more information on start mode see the source link.
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
+
+        <xs:element name="resetfailure" type="xs:string" minOccurs="0" maxOccurs="unbounded">
+          <xs:annotation>
+            <xs:documentation xml:lang="en">
+             This optional element controls the timing in which Windows SCM resets the failure count. 
+             For example, if you specify &lt;resetfailure&gt;1 hour &lt;/resetfailure&gt; and your service continues 
+             to run longer than one hour, then the failure count is reset to zero. This affects the 
+             behavior of the failure actions (see &lt;onfailure&gt; above).
+             In other words, this is the duration in which you consider the service has been running successfully. Defaults to 1 day.
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
+
+        <xs:element name="sleeptime" type="xs:string" minOccurs="0" maxOccurs="unbounded">
```

Comment:
I suggest removing `sleeptime` and `waithint` from docs as they don't need to be visible to users at all.